toriis-portal/toriis
# Cronjob to pull from ESG

## Summary
Creates a cronjob using github workflow (May switch over to vercel cronjobs later) to call the ESG API with 50 tickers a day from the mongodb database
Closes #5 
## Changes
- Added new api to backend
- Added new workflow to call api
- Added a new type for the Context


## Requests / Responses
To test the backend:
1.  Go to checks and rerun the cronjob in github actions. It should return success. 
2.  Go to the preview page [here](https://toriis-3qphhz1lf-secs.vercel.app/api/trpc/cronjob.esgpull). It should return an unauthorized message
